addons/product/models/product_template.py

```diff
@@ -1463,3 +1463,7 @@ def _get_contextual_pricelist(self):
         This method is meant to be overriden in other standard modules.
         """
         return self.env['product.pricelist'].browse(self.env.context.get('pricelist'))
+
+    def _get_gross_list_price(self, price):
```

Comment:
Suggestion: From what i'm seeing, the function will be override by other module so could be nice to have a docstring to explain in which condition we are using it and why we may need to override this ? :smile: